airflow-core/src/airflow/ui/public/i18n/locales/th/assets.json

```diff
@@ -0,0 +1,30 @@
+{
+  "consumingDags": "Dags ที่ใช้",
+  "createEvent": {
+    "button": "สร้างเหตุการณ์",
+    "manual": {
+      "description": "สร้าง Asset Event ด้วยตนเอง",
+      "extra": "ข้อมูลเพิ่มเติมของ Asset Event",
```

Comment:
I don't know Thai, but why did you decide to make "Asset Event" a proper noun? I think other translations only protected "Asset" but then translated the word "event"